ow_lander/src/ow_lander/transforms.py

```diff
@@ -0,0 +1,51 @@
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import tf2_ros
+
+from ow_lander.common import Singleton
+
+# NOTE: These imports are not directly used, but importing them works around a
+#       bug discussed here
+# https://answers.ros.org/question/249433/tf2_ros-buffer-transform-pointstamped/
+from tf2_geometry_msgs import *
+
+class FrameTransformer(metaclass = Singleton):
```

Comment:
Since this is the only class in the file I think the file should be named FrameTransformer.py. That's the convention we use for C++, but I don't see a similar convention written down in our python style guide. Do we have rules for naming python scripts?

Reply:
Neither our own or ROS's Python style guide outline guidelines for module file names, so I think it defer's to Python's [PEP8 style guide](https://peps.python.org/pep-0008/#naming-conventions).

Reply:
The name of the module is now `frame_transformer.py`

Reply:
Thanks. This is one thing I think C++ style does a lot better than Python, but I'll go with it for now.